scripts/build-probe-binaries

```diff
@@ -15,14 +15,14 @@ set -euo pipefail
 PROBE_NAME=$1
 PROBE_VERSION=$2
 REPOSITORY_NAME=$3
-KERNEL_TYPE=
+EXTRA_BUILD_INSTRU=
```

Comment:
why is this renaming required?

Reply:
it is not only the kernel type anymore; it is the kernel type + whether to containerize the build. to reflect the new purpose, hence change the name. another option (more structural) is to keep the KERNEL_TYPE as is, and add one more option names like BUILD_INSTRUCTION w/ values "container" and "process"; did not choose that as it requires one more optional arg, and the current arg seems able to express the purpose sufficiently well.

Reply:
i would prefer if we keep KERNEL_TYPE and have a separate arg called BUILD_TYPE.

Reply:
OK that makes sense. will change.